src/coreclr/vm/ClrEtwAll.man

```diff
@@ -3850,6 +3971,16 @@
                            task="GarbageCollection"
                            symbol="GenAwareEnd" message="$(string.RuntimePublisher.GenAwareEndEventMessage)"/>
 
+                    <event value="208" version="0" level="win:Informational"  template="GCLOHCompact"
+                           keywords ="GCKeyword"  opcode="GCLOHCompact"
+                           task="GarbageCollection"
+                           symbol="GCLOHCompact" message="$(string.RuntimePublisher.GCLOHCompactEventMessage)"/>
+
+                    <event value="209" version="0" level="win:Verbose"  template="GCFitBucketInfo"
```

Comment:
How verbose are these events relative to allocation_tick? I'm imagining we've got users that want to use Allocation_Tick for low overhead sampling, but because it is marked as a verbose event we are forcing them to receive all other GC verbose events even if they have no use for them.

Reply:
we already have other verbose level event in GC fired during a GC like the join events that are a lot more verbose than this. in fact we have informational level GC events that are fired more frequent than this one (eg, the GCMarkWithType is fired multiple times per heap and this event is fired once per heap during only gen1 GCs). and the users are already receiving them regardless whether they use them or not (often they don't since most tools don't interpret most of the GC events). 

overhead is relative - alloc tick is low overhead compared to other mechanisms (eg, it's very low ahead via the alloc monitoring with profiling API). for tracing I view it as medium overhead compared to events we _usually_ collect. certainly they are a lot more verbose than events fired during GCs.

Reply:
As long as the relative increase in event volume isn't large (comparing verbose before this PR to verbose afterwards) it should be fine : )